Test failure: `KeepAliveHandlerTest.testKeepAliveWithIdleTimeout()`

```
KeepAliveHandlerTest > testKeepAliveWithIdleTimeout() FAILED
    java.lang.AssertionError: 
    Expecting any element of:
      {"armeria.connections.lifespan#count"=0.0, "armeria.connections.lifespan#max"=0.0, "armeria.connections.lifespan#total"=0.0, "armeria.connections.lifespan.percentile#value{phi=0.5}"=0.0, "armeria.connections.lifespan.percentile#value{phi=0.75}"=0.0, "armeria.connections.lifespan.percentile#value{phi=0.95}"=0.0, "armeria.connections.lifespan.percentile#value{phi=0.98}"=0.0, "armeria.connections.lifespan.percentile#value{phi=0.999}"=0.0, "armeria.connections.lifespan.percentile#value{phi=0.99}"=0.0, "armeria.connections.lifespan.percentile#value{phi=0.9}"=0.0, "armeria.connections.lifespan.percentile#value{phi=0}"=0.0, "armeria.connections.lifespan.percentile#value{phi=1}"=0.0}
    to satisfy the given assertions requirements but none did:

    "armeria.connections.lifespan.percentile#value{phi=0.75}"=0.0
    error: 
    expected: "armeria.connections.lifespan#count"
     but was: "armeria.connections.lifespan.percentile#value{phi=0.75}"

    "armeria.connections.lifespan.percentile#value{phi=0.95}"=0.0
    error: 
    expected: "armeria.connections.lifespan#count"
     but was: "armeria.connections.lifespan.percentile#value{phi=0.95}"

    "armeria.connections.lifespan.percentile#value{phi=0.5}"=0.0
    error: 
    expected: "armeria.connections.lifespan#count"
     but was: "armeria.connections.lifespan.percentile#value{phi=0.5}"

    "armeria.connections.lifespan.percentile#value{phi=0.999}"=0.0
    error: 
    expected: "armeria.connections.lifespan#count"
     but was: "armeria.connections.lifespan.percentile#value{phi=0.999}"

    "armeria.connections.lifespan.percentile#value{phi=1}"=0.0
    error: 
    expected: "armeria.connections.lifespan#count"
     but was: "armeria.connections.lifespan.percentile#value{phi=1}"

    "armeria.connections.lifespan.percentile#value{phi=0.99}"=0.0
    error: 
    expected: "armeria.connections.lifespan#count"
     but was: "armeria.connections.lifespan.percentile#value{phi=0.99}"

    "armeria.connections.lifespan.percentile#value{phi=0.98}"=0.0
    error: 
    expected: "armeria.connections.lifespan#count"
     but was: "armeria.connections.lifespan.percentile#value{phi=0.98}"

    "armeria.connections.lifespan.percentile#value{phi=0}"=0.0
    error: 
    expected: "armeria.connections.lifespan#count"
     but was: "armeria.connections.lifespan.percentile#value{phi=0}"

    "armeria.connections.lifespan.percentile#value{phi=0.9}"=0.0
    error: 
    expected: "armeria.connections.lifespan#count"
     but was: "armeria.connections.lifespan.percentile#value{phi=0.9}"

    "armeria.connections.lifespan#count"=0.0
    error: 
    expected: 1.0
     but was: 0.0

    "armeria.connections.lifespan#total"=0.0
    error: 
    expected: "armeria.connections.lifespan#count"
     but was: "armeria.connections.lifespan#total"

    "armeria.connections.lifespan#max"=0.0
    error: 
    expected: "armeria.connections.lifespan#count"
     but was: "armeria.connections.lifespan#max"
        at com.linecorp.armeria.internal.common.KeepAliveHandlerTest.assertMeter(KeepAliveHandlerTest.java:507)
        at com.linecorp.armeria.internal.common.KeepAliveHandlerTest.testKeepAliveWithIdleTimeout(KeepAliveHandlerTest.java:341)
```

--- a/core/src/test/java/com/linecorp/armeria/internal/common/KeepAliveHandlerTest.java
+++ b/core/src/test/java/com/linecorp/armeria/internal/common/KeepAliveHandlerTest.java
@@ -126,7 +126,7 @@ class KeepAliveHandlerTest {
         ctx.channel().closeFuture().addListener(unused -> counter.incrementAndGet());
         await().timeout(4, TimeUnit.SECONDS).untilAtomic(counter, Matchers.is(2));
 
-        assertMeter(CONNECTION_LIFETIME + "#total", 1, withinPercentage(25));
+        await().untilAsserted(() -> assertMeter(CONNECTION_LIFETIME + "#total", 1, withinPercentage(25)));
         idleTimeoutScheduler.destroy();
     }
 
@@ -166,7 +166,7 @@ class KeepAliveHandlerTest {
         await().until(stopwatch::isRunning, Matchers.is(false));
         final Duration elapsed = stopwatch.elapsed();
         assertThat(elapsed.toMillis()).isBetween(1000L, 5000L);
-        assertMeter(CONNECTION_LIFETIME + "#count", 0);
+        await().untilAsserted(() -> assertMeter(CONNECTION_LIFETIME + "#count", 0));
         idleTimeoutScheduler.destroy();
     }
 
@@ -201,7 +201,7 @@ class KeepAliveHandlerTest {
         };
         keepAliveHandler.initialize(ctx);
 
-        assertMeter(CONNECTION_LIFETIME + "#count", 0);
+        await().untilAsserted(() -> assertMeter(CONNECTION_LIFETIME + "#count", 0));
         assertThat(keepAliveHandler.needToCloseConnection()).isFalse();
     }
 
@@ -338,7 +338,7 @@ class KeepAliveHandlerTest {
         assertThat(idleCounter).hasValue(0);
 
         await().timeout(idleTimeout * 2, TimeUnit.MILLISECONDS).untilAtomic(idleCounter, Matchers.is(1));
-        assertMeter(CONNECTION_LIFETIME + "#count", 1);
+        await().untilAsserted(() -> assertMeter(CONNECTION_LIFETIME + "#count", 1));
         idleTimeoutScheduler.destroy();
     }
 

